src/components/Notification/SbNotification.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div :class="computedClasses">
+    <SbIcon
+      class="sb-notification--icon-container"
+      size="small"
+      :name="icons[status]"
+    ></SbIcon>
+    <span v-if="title" class="sb-notification--title">
+      {{ capitalizeText(title) }}
+    </span>
+    <div v-if="isDescriptionVisible" class="sb-notification--description">
+      {{ descriptionToRender }}<slot></slot>
```

Comment:
For better readability it seems that `<slot></slot>` could be moved one line below